execution/executors/shim/process.go

```diff
@@ -35,8 +35,14 @@ func newProcess(ctx context.Context, o newProcessOpts) (*process, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer func() {
```

Comment:
Oh, I was misunderstanding that "Redeclaration does not introduce a new variable; it just assigns a new value to the original.". :sweat_smile: 
https://golang.org/ref/spec#Short_variable_declarations
https://play.golang.org/p/3T1ZVYZ8wa

I'll update PR in a short..